packages/squiggle-lang/src/library/registry/index.ts

```diff
@@ -29,6 +29,7 @@ import { ImmutableMap } from "../../utility/immutableMap.js";
 
 const fnList: FRFunction[] = [
   ...builtinLibrary,
+  ...numberLibrary,
```

Comment:
This is important so that Number.sum() gets run before Dist.sum(). Neither requires a namespace. Dist.sum() converts numbers to dists, so it's important that Number.sum() gets checked first.

This ordering is awkward. The main alternative is to have both definitions in Number, but this felt awkward for other reasons. (We could also require the use of the Namespace for the Dist* methods)

Reply:
Can you add a comment? I can see myself sorting this list in the future, or reorganize it somehow. (But I guess `sum[]` tests will catch that?)